Samples/Forms/LibVLCSharp.Forms.MediaElement/LibVLCSharp.Forms.MediaElement.Sample/MainViewModel.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Windows.Input;
+using GalaSoft.MvvmLight;
+using GalaSoft.MvvmLight.Command;
+using LibVLCSharp.Shared;
+using Xamarin.Forms;
+
+namespace LibVLCSharp.Forms.Sample.MediaPlayerElement
+{
+    /// <summary>
+    /// Represents the main viewmodel.
+    /// </summary>
+    public class MainViewModel : ViewModelBase
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="MainViewModel"/> class.
+        /// </summary>
+        public MainViewModel()
+        {
+            InitCommand = new RelayCommand(Init);
+        }
+
+        private LibVLC _libVLC;
+        /// <summary>
+        /// Gets the <see cref="LibVLCSharp.Shared.LibVLC"/> instance.
+        /// </summary>
+        public LibVLC LibVLC
+        {
+            get => _libVLC;
+            private set => Set(nameof(LibVLC), ref _libVLC, value);
+        }
+
+        private MediaPlayer _mediaPlayer;
+        /// <summary>
+        /// Gets the <see cref="LibVLCSharp.Shared.MediaPlayer"/> instance.
+        /// </summary>
+        public MediaPlayer MediaPlayer
+        {
+            get => _mediaPlayer;
+            private set => Set(nameof(MediaPlayer), ref _mediaPlayer, value);
+        }
+
+        /// <summary>
+        /// Gets the initialization command.
+        /// </summary>
+        public ICommand InitCommand { get; }
+
+        private void Init()
+        {
+            Core.Initialize();
+
+            LibVLC = new LibVLC();
+
+            var media = new Media(LibVLC,
+                "http://download.blender.org/peach/bigbuckbunny_movies/big_buck_bunny_480p_surround-fix.avi",
+                FromType.FromLocation);
+
+            if(Device.RuntimePlatform == Device.Android)
```

Comment:
Wasn't the purpose of `MediaConfiguration.EnableHardwareDecoding` to be cross-platform and not requiring such tests? Can we remove it here?

Reply:
Yes